app/src/main/java/org/p2p/wallet/home/ui/container/MainContainerPresenter.kt

```diff
@@ -179,9 +179,9 @@ class MainContainerPresenter(
             is UserTokensState.Loaded -> {
                 // todo: this new filter supposed to be used for new design
                 val filteredTokens = newState.solTokens.filterTokensForWalletScreen()
-                val balance = filteredTokens.sumOf { it.total }
-                view?.showWalletBalance(balanceMapper.formatBalance(balance))
-                view?.showWalletBalance("Wallet")
+                val balance = filteredTokens.sumOf(Token.Active::total)
+                view?.showWalletBalance(balance = balanceMapper.formatBalance(balance))
+                view?.showWalletBalance(balance = "Wallet")
```

Comment:
one of these calls is not needed

Reply:
won't be touching that, it's a wallet screen that is not used